migtests/tests/pg/indexes/pg_indexes_automation.sql

```diff
@@ -29,7 +29,7 @@ SELECT
   md5(random()::text),
   md5(random()::text)
 from (
-  SELECT * FROM generate_series(1,100000) AS id
+  SELECT * FROM generate_series(1,1000) AS id
```

Comment:
1. reasoning behind this?
2. How are you going to verify this fixes the issue? run it multiple times?